src/Method/Handler/RegisterHandlerInterface.php

```diff
@@ -9,7 +9,7 @@
  * Represents the base requirements for implementing an MFA Method's RegisterHandler, which has the responsibility
  * of initiating and managing registration of the MFA Method in question against the current Member.
  */
-interface RegisterHandlerInterface
+interface RegisterHandlerInterface extends UIComponentProviderInterface
```

Comment:
My gut feeling after looking at this change for all of 8 seconds is that we should keep these separate and implement both of them explicitly

Reply:
Yeah I figured this might be a little controversial. I'm happy to remove this interface.